src/rdkafka_offset.c

```diff
@@ -173,7 +173,7 @@ static int64_t rd_kafka_offset_file_read (rd_kafka_toppar_t *rktp) {
 	char buf[22];
 	char *end;
 	int64_t offset;
-	int r;
+	size_t r;
```

Comment:
the fread() below should probably be changed to:
if ((r = fread(buf, 1, ...)) < 1) {  rather than == -1